diff --git a/src/meson.build b/src/meson.build
index 5d64188..4631a2c 100644
--- a/src/meson.build
+++ b/src/meson.build
@@ -7,7 +7,7 @@ executable('Hyprland', src,
     server_protos,
     dependency('wayland-server'),
     dependency('wayland-client'),
-    wlroots.get_variable('wlroots'),
+    dependency('wlroots'),
     dependency('cairo'),
     dependency('pango'),
     dependency('pangocairo'),
