diff --git a/meson.build b/meson.build
index 971be7e6..6652b06c 100644
--- a/meson.build
+++ b/meson.build
@@ -15,7 +15,7 @@ project(
 # necessary for bugfix releases. Increasing soversion is required because
 # wlroots never guarantees ABI stability -- only API stability is guaranteed
 # between minor releases.
-soversion = 11
+soversion = 11032
 
 little_endian = target_machine.endian() == 'little'
 big_endian = target_machine.endian() == 'big'
